src/docs/Foundations/MagicUnit/code.mdx

```diff
@@ -79,21 +79,21 @@ We created multiple things to help you work better with the magic unit :
 
 here their respectives API :
 
-### The magic-unit function :
+### The magic-unit function
 
-> the **magic-unit function** return an **unitless** multiplictaion of the magic unit **expressed in px**, and expect a **multiplier** parameter.
+> The **magic-unit function** return an **unitless** multiplictaion of the magic unit **expressed in px**, and expect a **multiplier** parameter.
```

Comment:
The **magic-unit function** returns a **unitless** multiplication of the magic unit **expressed in px**, and expects a **multiplier** parameter.

Reply:
fixed